src/win_tools/ctrlservice/ctrlservice.cpp

```diff
@@ -83,6 +83,7 @@ int
 _tmain (int argc, char *argv[])
 {
   bool rc;
+  int broker_start_loop_cnt_max = 40;
```

Comment:
How about setting it to maximum 10 seconds? That is, changing the value from 40 to 50.

Reply:
Okay, I'll change that.